packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -146,6 +146,7 @@ export function TopNavImpl(props: Props) {
         disabledOverflow
         selectedKeys={[pathname]}
         items={itemsGlobalRight}
+        style={{ color: '#fff' }}
```

Comment:
Shouldn't this be in css somewhere?

Reply:
Yes, but, since the change was a one-liner, I thought of adding an inline style. Moving it to the CSS file.